Gruntfile.js

```diff
@@ -130,91 +111,17 @@ module.exports = function( grunt ) {
           src: [
             "public/editor/**/*.js",
             "public/homepage/**/*.js",
+            "public/resources/remix/index.js",
```

Comment:
I think this one is worth doing, since it's the basis of our injected remix header, and if it has obvious JS issues, it would be nice to get those fixed in development before we ship.  I agree with not having everything in `public/resources/*` being linted--we can add more to this list when we add things we write.